2021-12-07 20:13:41,418 INFO    MainThread:382031 [wandb_setup.py:_flush():71] setting env: {}
2021-12-07 20:13:41,418 INFO    MainThread:382031 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-07 20:13:41,419 INFO    MainThread:382031 [wandb_init.py:_log_setup():371] Logging user logs to /home/beegass/Documents/Coding/Readable-VAEs/vae-pytorch/wandb/run-20211207_201341-3fis55ej/logs/debug.log
2021-12-07 20:13:41,419 INFO    MainThread:382031 [wandb_init.py:_log_setup():372] Logging internal logs to /home/beegass/Documents/Coding/Readable-VAEs/vae-pytorch/wandb/run-20211207_201341-3fis55ej/logs/debug-internal.log
2021-12-07 20:13:41,419 INFO    MainThread:382031 [wandb_init.py:init():404] calling init triggers
2021-12-07 20:13:41,419 INFO    MainThread:382031 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-07 20:13:41,419 INFO    MainThread:382031 [wandb_init.py:init():449] starting backend
2021-12-07 20:13:41,419 INFO    MainThread:382031 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-07 20:13:41,429 INFO    MainThread:382031 [backend.py:ensure_launched():199] starting backend process...
2021-12-07 20:13:41,436 INFO    MainThread:382031 [backend.py:ensure_launched():204] started backend process with pid: 382110
2021-12-07 20:13:41,437 INFO    MainThread:382031 [wandb_init.py:init():458] backend started and connected
2021-12-07 20:13:41,438 INFO    MainThread:382031 [wandb_init.py:init():519] updated telemetry
2021-12-07 20:13:41,453 INFO    MainThread:382031 [wandb_init.py:init():549] communicating current version
2021-12-07 20:13:42,261 INFO    MainThread:382031 [wandb_init.py:init():554] got version response 
2021-12-07 20:13:42,261 INFO    MainThread:382031 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-07 20:13:42,642 INFO    MainThread:382031 [wandb_init.py:init():592] starting run threads in backend
2021-12-07 20:13:42,674 INFO    MainThread:382031 [wandb_run.py:_console_start():1816] atexit reg
2021-12-07 20:13:42,674 INFO    MainThread:382031 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-07 20:13:42,675 INFO    MainThread:382031 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-07 20:13:42,676 INFO    MainThread:382031 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-07 20:13:42,676 INFO    MainThread:382031 [wandb_init.py:init():619] run started, returning control to user process
2021-12-07 20:33:54,067 INFO    MainThread:382031 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2021-12-07 20:33:54,068 INFO    MainThread:382031 [wandb_run.py:_restore():1758] restore
2021-12-07 20:33:56,486 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 756
}

2021-12-07 20:33:56,594 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 756
}

2021-12-07 20:33:56,982 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 756
}

2021-12-07 20:33:57,084 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,185 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,287 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,389 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,491 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,592 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,694 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 756
  total_bytes: 8351
}

2021-12-07 20:33:57,796 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:57,898 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:58,000 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:58,101 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:58,203 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:58,305 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:58,519 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}

2021-12-07 20:33:59,417 INFO    MainThread:382031 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8351
  total_bytes: 8351
}
local_info {
}

2021-12-07 20:34:00,509 INFO    MainThread:382031 [wandb_run.py:_append_files():2184] logging synced files
